builtin/logical/database/rotation_test.go

```diff
@@ -2,6 +2,8 @@ package database
 
 import (
 	"context"
+	"go.mongodb.org/mongo-driver/mongo"
```

Comment:
nit: these should be in the section below, and the imports in general could benefit from a `gofmt`

Reply:
Thanks! Went through all files and made sure `goimports` and `gofmt` are applied.